recipes/cogger/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "cogger" %}
+{% set version = "0.0.7" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/airbusgeo/cogger/archive/v${{ version }}.tar.gz
```

```suggestion
    url: https://github.com/airbusgeo/cogger/archive/v{{ version }}.tar.gz
```
I made a mistake in the previous suggestion - this is what it's supposed to be.